k8s.go

```diff
@@ -27,6 +27,7 @@ type Cert string
 const (
 	APICert            Cert = "api"
 	CalicoCert         Cert = "calico"
+	ChartOperatorCert  Cert = "chart-operator"
```

Comment:
Question: Should we have separate certs for chart-operator and cluster-operator?

The cluster-operator access is to install chart-operator. We will also need certs for chart-operator when it runs inside the guest cluster.

Reply:
When chart operator runs inside the guest cluster it can simply use the in-cluster config like every other service. Auth is then done via RBAC/PSP.

Reply:
So here we only have to sort the cluster-operator AFAIK.

Reply:
Ah good point, thanks! I'll change to use cluster-operator.